src/app/utils/Validators.ts

```diff
@@ -39,18 +40,77 @@ export class DateValidators {
 
     if (!isNaN(fechaDate.getTime())) {
       const año: number = fechaDate.getFullYear();
-      const mes:string = fechaDate.toLocaleString('es-ES', {month: 'long' })
-      const mesMayuscula = mes.charAt(0).toUpperCase() + mes.slice(1) ;
-
-
-
+      const mes: string = fechaDate.toLocaleString('es-ES', { month: 'long' });
+      const mesMayuscula = mes.charAt(0).toUpperCase() + mes.slice(1);
       const dia: number = fechaDate.getDate() + 1;
-
-      const fechaFormateada: string = `${dia.toString().padStart(2, '0')} - ${mesMayuscula} - ${año}`;
+      const fechaFormateada: string = `${dia
+        .toString()
+        .padStart(2, '0')} - ${mesMayuscula} - ${año}`;
       return fechaFormateada;
     } else {
       console.error('La cadena de fecha no es válida.');
       return '';
     }
   }
 }
+
+export class NormaliceLowerValidators {
+  static normaliceData(data: any): void {
+    for (const key in data) {
+      if (data.hasOwnProperty(key)) {
+        if (key !== 'password') {
+          if (typeof data[key] === 'string') {
+            data[key] = data[key].toLowerCase();
+          }
```

Comment:
compa mejor prueba con este pedazo de codigo

`export class NormaliceLowerValidators {
  static normaliceData(data: Record<string, any>): void {
    const { password, ...rest } = data;

    for (const key of Object.keys(rest)) {
      if (typeof rest[key] === 'string') {
        rest[key] = rest[key].toLowerCase();
      }
    }
  }
}`

eso esta muy anidado.